pxr/usdImaging/usdImaging/delegate.h

```diff
@@ -659,7 +659,13 @@ class UsdImagingDelegate : public HdSceneDelegate, public TfWeakBase {
     _DependencyMap _dependencyInfo;
 
     void _GatherDependencies(SdfPath const& subtree,
-                             SdfPathVector *affectedCachePaths);
+                             SdfPathVector& affectedCachePaths);
+
+    typedef TfHashMap<SdfPath, SdfPathVector, SdfPath::Hash> _DependenciesCacheMap;
+    _DependenciesCacheMap _dependenciesCacheMap;
+
+    void _GatherDependenciesCache(SdfPath const &subtree,
```

Comment:
Can we change this to _CacheDependencies or _PrecacheDependencies?  It makes it more clear what the function is doing.

The pixar code convention is that parameters a function writes out to have to be a pointer, not a reference.  Since the SdfPathVector is owned by the _dependenciesCacheMap, you could change this to SdfPathVector**, or else make the return type "const SdfPathVector&" instead of void...